src/core/main.cpp

```diff
@@ -403,6 +403,28 @@ int main( int argc, char * * argv )
 
 			return EXIT_SUCCESS;
 		}
+		else if (arg == "makeBundle" || arg == "--makeBundle")
+		{
+			++i;
+
+			if (i == argc)
+			{
+				return noInputFileError();
+			}
+
+			DataFile dataFile(QString::fromLocal8Bit(argv[i]));
+
+			if (argc > i+1) // Project bundle file name given
```

Comment:
Isn't that true for the `upgrade` action as well? I just copied the way it handles obtaining the input and output files from it IIRR. If we need that level of sanitization maybe we should leave it for a separate issue to fix the other actions as well?

Reply:
Hmm it would be better to check these things, but you can do it later if you want. Or fix it now on master and then rebase your code and fix it on your branch. I leave it to you.

Reply:
Could we do it later? Then we can think of how we should sanitize it both for `upgrade` and `makebundle` and fix both at once. We should fix it but it's not that urgent I think, since few users actually use the CLI and those will usually be more experienced to know that that kind of input could go wrong :smile:

Reply:
OK. resolving this comment.